xobjects/context.py

```diff
@@ -679,7 +684,7 @@ def get_test_contexts():
             yield xo.ContextPyopencl()
```

Comment:
I suggest to add a OpenMP test context to the default test contexts (uncomment line 680 and maybe use "auto" instead of 2)